src/vehicle.cpp

```diff
@@ -3694,6 +3697,84 @@ bool vehicle::sufficient_wheel_config() const
     return true;
 }
 
+bool vehicle::handle_potential_theft( player *p, bool check_only, bool prompt )
+{
+    if( p == nullptr ) {
+        return false;
+    }
+    faction *yours;
+    if( p->is_player() ) {
+        yours = g->faction_manager_ptr->get( faction_id( "your_followers" ) );
+    } else {
+        npc *guy = dynamic_cast<npc *>( p );
+        yours = guy->my_fac;
+    }
+    std::vector<npc *> witnesses;
+    for( npc &elem : g->all_npcs() ) {
+        if( rl_dist( elem.pos(), p->pos() ) < MAX_VIEW_DISTANCE && get_owner() && elem.my_fac == owner &&
```

Comment:
Why are you using the accessor function (`get_owner`) here in a boolean context, you explicitly added a `has_owner` for that purpose, why are you not using it.

In the same way, you are accessing `vehicle::owner` directly here, instead of using the accessor function.

Nothing of that is wrong, it's just inconsistent and confusing (was it supposed to be that way? would using one of the accessor functions change the behavior? why?).